### PR TITLE
Update evaluate.py

### PR DESCRIPTION
**Pull request by eurusebr**
I made the changes to the nav: int=10 | None. I also updated the nav in the parameters section.

### CODE REVIEW

File: amep/evaluate.py [hunk 1]
```diff
@@ -3678,8 +3682,9 @@ def __init__(
             Skip this fraction at the beginning of the trajectory.
             The default is 0.0.
         nav : int, optional
-            Total number of time steps (frames) to consider.
-            The default is None.
+            Number of frames to consider for the time average.
+            The default is 10 if None, i.e., all
```
Comment (kay-ro) [inline amep/evaluate.py:3686]: remove "if None, i.e., all available time steps are used." This option is only available for a few classes.
We want users to have a consistent experience. to use all time-steps they should give "traj.nframes" or np.inf if they are sketchy... ;)

File: amep/evaluate.py [hunk 2]
```diff
@@ -3997,7 +4002,7 @@ def __init__(
             of the trajectory. The default is 0.0.
         nav : int or None, optional
             Number of time steps at which the mean square
-            displacement should be evaluated. The default is None, i.e., all 
+            displacement should be evaluated. The default is 10 if None, i.e., all
```
Comment (kay-ro) [inline amep/evaluate.py:4005]: remove "if None, i.e., all available time steps are used."

File: amep/evaluate.py [hunk 3]
```diff
@@ -4221,7 +4226,8 @@ def __init__(
             of the trajectory. The default is 0.0.
         nav : int, optional
             Number of time steps at which the autocorrelation
-            function should be evaluated. The default is None.
+            function should be evaluated. The default is 10 if None, i.e., all
```
Comment (kay-ro) [inline amep/evaluate.py:4229]: remove "if None, i.e., all available time steps are used."

File: amep/evaluate.py [hunk 4]
```diff
@@ -4416,7 +4422,8 @@ def __init__(self, traj, ptype=None, skip=0.0, nav=None, direction='xyz'):
...
```
Comment (kay-ro) [inline amep/evaluate.py:4425]: remove "if None, i.e., all available time steps are used."

File: amep/evaluate.py [hunk 5]
```diff
@@ -4636,7 +4643,8 @@ def __init__(self, traj, *args, ptype=None, skip=0.0, nav=None):
...
```
Comment (kay-ro) [inline amep/evaluate.py:4646]: remove "if None, i.e., all available time steps are used."

**Review by kay-ro — CHANGES_REQUESTED** [review]
please change the small comments.
- remove "if None, i.e., all available time steps are used."

File: amep/evaluate.py (hunk 1 above)
Comment (eurusebr) [reply amep/evaluate.py]: Done.

File: amep/evaluate.py (hunk 2 above)
Comment (eurusebr) [reply amep/evaluate.py]: Done.

File: amep/evaluate.py (hunk 3 above)
Comment (eurusebr) [reply amep/evaluate.py]: Done.

File: amep/evaluate.py (hunk 5 above)
Comment (eurusebr) [reply amep/evaluate.py]: Done! :)

**Review by kay-ro — APPROVED** [review]
looks good. Thank you :)

**Pull request merged**